site/docs/design_tokens/box-shadow.mdx

```diff
@@ -0,0 +1,32 @@
+---
+name: Box shadow
+menu: Design tokens
+route: /design-tokens/box-shadow
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+# Box shadow
+
+<LargeParagraph>Shadows can be used under certain elements to create visual distinction for content that otherwise would be hard to distinguish for the visually impaired or remain unnoticed.
+</LargeParagraph>
+
+Box-shadows are used to cast shadows from interface elements to backgrounds or other elements to allow visual cohesion and a more accessible experience.
+
+## Principles
+- **Shadows are not used to signify elevation in HDS.** Elements that require distinction, can however have shadows applied.
+- **Do not use shadows to highlight elements.** Elements with shadow around them should only have it for visual clarity.
```

Comment:
> Do not use shadows to highlight elements.** Elements with shadow around them should only have it for visual clarity.

Could this be elaborated more? Maybe add two examples as a sublist; an example what is for visual clarity and an example for highlighting?

Reply:
- [x] Small fixes and additions to the Shadow box token documentation